### PR TITLE
Add sample code

### PR DESCRIPTION
**Pull request by vietanh-epi**
A simple console application
Some readme files

### CODE REVIEW

File: Course-CSharp/VietAnh/ConsoleAppEpi/ConsoleAppEpi/Program.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            Console.WriteLine("Nhap vao so B:");
+            string b = Console.ReadLine();
+
+            int numberA = int.Parse(a);
+            int numberB = int.Parse(b);
+
+            var calculator = new Calculator();
+
+            Console.WriteLine("Ket qua:");
+            Console.WriteLine(calculator.Sum(numberA, numberB));
```
Comment (vietanh-epi) [inline Course-CSharp/VietAnh/ConsoleAppEpi/ConsoleAppEpi/Program.cs]: Should be
Console.WriteLine("Ket qua cua phep Cong: " + calculator.Sum(numberA, numberB))

File: Course-CSharp/VietAnh/ConsoleAppEpi/ConsoleAppEpi/Calculator.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+namespace ConsoleAppEpi
+{
+    class Calculator
+    {
+        public int Sum(int a, int b)
+        {
+            return a + b;
+        }
+
```
Comment (ldkl8778) [inline Course-CSharp/VietAnh/ConsoleAppEpi/ConsoleAppEpi/Calculator.cs]: abc

Comment (vietanh-epi) [reply Course-CSharp/VietAnh/ConsoleAppEpi/ConsoleAppEpi/Calculator.cs]: ok I'll update

File: Course-CSharp/VietAnh/ConsoleAppEpi/ConsoleAppEpi/Calculator.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```
Comment (xwind1999) [inline Course-CSharp/VietAnh/ConsoleAppEpi/ConsoleAppEpi/Calculator.cs]: You can remove this.

File: Course-CSharp/VietAnh/ConsoleAppEpi/ConsoleAppEpi/Calculator.cs (hunk 2 above)
Comment (vietanh-epi) [reply Course-CSharp/VietAnh/ConsoleAppEpi/ConsoleAppEpi/Calculator.cs]: Done

**Pull request merged**